src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -249,9 +238,9 @@ private void launchInitialNameNodes(SchedulerDriver driver, Collection<Offer> of
       if (offers.size() > 0) {
         Offer offer = offers.iterator().next();
         pendingOffers.remove(offer.getId());
-        launchNode(driver, offer, NAME_NODE_ID,
-                 Arrays.asList(NAME_NODE_ID, ZKFC_NODE_ID, JOURNAL_NODE_ID),
-                 NAME_NODE_EXECUTOR_ID);
+        launchNode(driver, offer, HDFSConstants.NAME_NODE_ID,
+            Arrays.asList(HDFSConstants.NAME_NODE_ID, HDFSConstants.ZKFC_NODE_ID,
+            HDFSConstants.JOURNAL_NODE_ID), HDFSConstants.NAME_NODE_EXECUTOR_ID);
```

Comment:
I'm not a big fan of this indentation style, since the parameters of Arrays.asList intermingle with the parameters of launchNode, with no difference in indentation. In Mesos, we would do
```
launchNode(
    driver, 
    offer,
    HDFSConstants.NAME_NODE_ID,
    Arrays.asList(HDFSConstants.NAME_NODE_ID, HDFSConstants.ZKFC_NODE_ID,
        HDFSConstants.JOURNAL_NODE_ID),
    HDFSConstants.NAME_NODE_EXECUTOR_ID);
```